blocks/extensions/plot.py

```diff
@@ -4,14 +4,12 @@
 from subprocess import Popen, PIPE
 
 try:
-    from bokeh.plotting import (curdoc, cursession, figure, output_server,
```

Comment:
curdoc was used by `__setstate__`, while cursession() is now unused because I use self.session. This prevents problems when you have multiple extensions plotting (i.e. multiple sessions open)